core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreManager.java

```diff
@@ -124,39 +133,21 @@ public static KeyStoreManager getInstance(int tenantId,
      * @return KeyStore object
      * @throws Exception If there is not a key store with the given name
      */
-    public KeyStore getKeyStore(String keyStoreName) throws Exception {
+    public KeyStore getKeyStore(String keyStoreName) {
```

Comment:
Don't change the public method definitions

Reply:
Fixed in https://github.com/wso2/carbon-kernel/pull/4025/commits/c1687dc983cee4418f175db98e13c1744c08a510